src/Table/tests/Table.static.test.js

```diff
@@ -136,15 +137,16 @@ describe('Table', () => {
       })
 
       cases('should render headercolumns with correct titles',
-        ({ component, columns }) => {
+        ({ columns, component }) => {
           const header = component.find(TableHead)
           const headerColumns = header
             .find('th')
-            .findWhere(node =>
-              node.length > 0 && node.type() === 'div' &&
-              node.find('span').length > 0 &&
-              node.find('input').lenght <= 0
-            )
+            .findWhere(node => (
+              node.length > 0
+              && node.type() === 'div'
+              && node.find('span').length > 0
+              && node.find('input').lenght <= 0
```

Comment:
typo, but it already exists in the previous code 
```suggestion
  && node.find('input').length <= 0
```